<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TV9Q0GU" failures="2" tests="4" name="BankingApp Test(failed)" time="52.245" errors="0" timestamp="11 Mar 2021 20:19:05 GMT">
  <testcase classname="com.inetBanking.testCases.TC_LoginTest_001" name="loginTest" time="4.198"/>
  <testcase classname="com.inetBanking.testCases.TC_AddCustomerTest_003" name="addNewCustomer" time="14.313"/>
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="5.617">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetBanking.testCases.TC_LoginDDT_002" name="loginDDT" time="5.891">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed) -->
